test/e2e/atlas/streams_test.go

```diff
@@ -300,6 +303,61 @@ func TestStreams(t *testing.T) {
 		a.Equal("SSL", connection.Security.GetProtocol())
 	})
 
+	t.Run("Creating a streams connection with atlas cluster", func(t *testing.T) {
+		clusterName := "Cluster1"
+
+		cmd := exec.Command(cliPath,
```

Comment:
currently this test runs on PRs a rule we don't run tests on PRs that create clusters (they are slow they + cost money) you can move this whole suite out of the PR checks or move only this test to a new suite